data_steward/validation/main.py

```diff
@@ -502,6 +477,38 @@ def get_drug_class_counts_query(hpo_id):
     return render_query(consts.DRUG_CHECKS_QUERY_VALIDATION, table_id=table_id)
 
 
+def get_hpo_missing_pii_query(hpo_id):
+    """
+    Query to retrieve counts of drug classes in an HPO site's drug_exposure table
+
+    :param hpo_id: identifies the HPO site
+    :return: the query
+    """
+    person_table_id = bq_utils.get_table_id(hpo_id, common.PERSON)
+    pii_name_table_id = bq_utils.get_table_id(hpo_id, common.PII_NAME)
+    pii_wildcard = bq_utils.get_table_id(hpo_id, common.PII_WILDCARD)
+    participant_match_table_id = bq_utils.get_table_id(hpo_id,
+                                                       common.PARTICIPANT_MATCH)
+    rdr_dataset_id = bq_utils.get_rdr_dataset_id()
+    rdr_date = rdr_dataset_id[3:]
```

Comment:
These indices are a bit sketchy - I'd probably use a regex or at least make an obvious assertion about the input value, e.g. throw/log an error if it doesn't start with the expected prefix, or isn't the correct length. Also worth adding an example value in a comment here to make this clear

Reply:
Yeah, good point. I'll add something here to check for the format and test it.